build_tools/llvm_version.txt

```diff
@@ -1 +1 @@
-e750c41ec1accc25f12e46fd64e52cd1495cfc58
+c1864ab9534080ee77a016bca24dd9a318bc6d7e
```

Comment:
rename the PR with the new SHA, or it will be confusing when we see that as a commit message